.github/PULL_REQUEST_TEMPLATE/bugfix.md

```diff
@@ -0,0 +1,25 @@
+Fixes: #<issue>
+
+## Description
+
+Please, detail the root cause of the bug and how this PR fixes it.
+
+## PR review check list
+
+Quality check list that cannot be automatically verified.
```

Comment:
This is to be filled in by the reviewer and taken into account by the author, right?

Reply:
Yes, that is what I understood from the `Specification` section in the issue https://github.com/raiden-network/raiden/issues/3066#issue-384330421

Reply:
Great. Makes sense.